testutils/application.go

```diff
@@ -0,0 +1,85 @@
+package testutils
+
+import (
+	"github.com/anchore/clio"
+	"github.com/google/go-cmp/cmp"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"sync"
+	"testing"
+)
+
+// TODO: is this needed given WrapForTesting? We need to think about
+// how clio is handling global state.
+// in particular, the testing pattern can cause initializers to be called more than once.
+func NewForTesting(t *testing.T, cfg *clio.SetupConfig, assertions ...AssertionFunc) clio.Application {
```

Comment:
nit on name: maybe `NewApplication` so that it reads `cliotestutils.NewApplication`